packages/validator/src/services/indices.ts

```diff
@@ -101,7 +101,13 @@ export class IndicesService {
     ApiError.assert(res, "Can not fetch state validators from beacon node");
 
     const newIndices = [];
+
+    const allValidators = new Map<ValidatorStatus, ValidatorResponse[]>();
+
     for (const validatorState of res.response.data) {
+      // Group all validators by status
+      allValidators.set(validatorState.status, [validatorState]);
+
       const pubkeyHex = toHexString(validatorState.validator.pubkey);
       if (!this.pubkey2index.has(pubkeyHex)) {
         this.logger.debug("Discovered validator", {pubkey: pubkeyHex, index: validatorState.index});
```

Comment:
This log should be replaced with the "Validator exists in beacon chain" log.
```suggestion
        this.logger.debug("Validator exists in beacon chain", {validatorIndex: validatorState.index, feeRecipient, pubkey: pubkeyHex});
```